jobs/service-fabrik-broker/templates/bin/health_check.erb

```diff
@@ -7,16 +7,16 @@
 # we will return success from here only if the current host is MASTER
 
 # check if internal IP is accessible
-nc -z <%= p('internal.ip') %> <%= p('external.port') %>
+nc -z <%= spec.ip %> <%= p('external.port') %>
```

Comment:
maybe remove/modify comment as it could be misleading.

Reply:
Done.